node/follower_node.go

```diff
@@ -414,7 +407,15 @@ func (node *AlgorandFollowerNode) SetCatchpointCatchupMode(catchpointCatchupMode
 			prevNodeCancelFunc()
 			return
 		}
+
+		// Catchup finished, resume.
 		defer node.mu.Unlock()
+
+		// update sync round before starting services
+		if err := node.SetSyncRound(uint64(node.ledger.LastRound())); err != nil {
+			node.log.Warnf("unable to set sync round while resuming fast catchup: %v", err)
+		}
```

Comment:
We'll need to poll the status of catchpoint catchup to see when it finishes so that we can set the proper sync round again in Conduit.

For example, if we want to start at round 1010, but our catchpoint is at round 1000, we'll now need to catchpoint catchup, wait for completion, then set the sync round to 1010 and wait for regular catchup.